cdap-ui/app/features/adapters/routes.js

```diff
@@ -64,7 +70,8 @@ angular.module(PKG.name + '.feature.adapters')
             templateUrl: '/assets/features/adapters/templates/tabs/runs.html',
             controller: 'AdapterRunsController',
             ncyBreadcrumb: {
-              label: 'Runs'
+              parent: 'adapters.list',
+              label: '{{$state.params.adapterId | caskCapitalizeFilter}}'
```

Comment:
should we not capitalize the adapter Id?
Since there can be two different adapters, such as `myadapter` and `myAdapter`.